estate/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "E-State!",
+
+    'summary': """
+        Manage real estate advertisements
+    """,
+
+    'description': """
+        Manage advertisements and offers for different real estate properties as part of the "Sever framework 101" tutorial
+    """,
+
+    'author': "mamr",
```

Comment:
```suggestion
    'author': 'Odoo',
```

Usually, we don't add the author on modules.
If you really want to keep it clean, please use "Odoo" (or "Odoo SA", ...) as your work for Odoo.